notify.js

```diff
@@ -99,6 +108,11 @@
         try {
             new N('');
         } catch (e) {
+            if (SW) {
```

Comment:
Well, this is the best I could come up with if we want a synchronous function... I agree with you, it's not perfect
Can totally just use `w.navigator.serviceWorker`, I just thought I'd try to follow your previous conventions